tutorials/quickstart/README.md

```diff
@@ -74,70 +86,151 @@ INFO    22-Apr 13:34:26 __init__ 9988 <Ctrl-B> opens the browser. <Ctrl-D> start
 
 <img src="https://cloud.gramener.com/f/5684de979acd45d4a14d/?dl=1" width="400"></img>
-</details>
 
-## Step 1: Expose the data through a REST API
-<details>
-  <summary> Expand This Section </summary>
-
-In order to provide our dashboard with access to the data, we need to create a URL that sends data to the dashboard. To do this, we use a Gramex component called [`FormHandler`](../formhandler).
 
-<script type="text/html" src="snippets/call-to-action-cards.html" class="step1gramex"></script>
+## Step 1: Expose the data through a REST API
 
+In order to provide our dashboard with access to the data,
+we need to create a URL that sends data to the dashboard. To do this,
+we use a Gramex component called [`FormHandler`](../formhandler).
 
-After you save the file, Gramex will be able to serve the CSV data through the `/data` resource endpoint. To verify this, visit [`http://localhost:9988/data?_limit=10`](http://localhost:9988/data?_limit=10) in your browser. You should now see a JSON payload representing the first ten rows of the dataset.
+Create a formhandler endpoint by adding the following lines to the empty <kbd>gramex.yaml</kbd> file,
+which we had created in the previous section:
 
-<script type="text/html" class="step1dataop" src="snippets/example-output.html"></script>
+```yaml
+url:
+  superstore-data:
+    pattern: /$YAMLURL/data
+    handler: FormHandler
+    kwargs:
+      url: $YAMLPATH/store-sales.csv
+```
 
 
-You could also, visit [http://localhost:9988/data?_limit=10&_format=html](http://localhost:9988/data?_limit=10&_format=html) to see the first ten rows as a simple HTML table.
+After you save the file, Gramex will be able to serve the CSV data through the
+`/data` resource endpoint.
+To verify this, visit
+[`http://localhost:9988/tutorials/data?_limit=10`](http://localhost:9988/tutorials/data?_limit=10)
+in your browser. You should now see a JSON payload representing the
+first ten rows of the dataset as follows:
+
+```js
+[
+  {
+    "Order ID":"CA-2016-152156","Order Date":"08-11-2016","Ship Date":"11-11-2016",
+    "Ship Mode":"Second Class","Customer Name":"Claire Gute","Segment":"Consumer",
+    "City":"Henderson","State":"Kentucky","Region":"South","Category":"Furniture",
+    "Sub-Category":"Bookcases","Sales":261.96,"Quantity":2,"Discount":0.0,"Profit":41.9136
+  },
+  {
+    "Order ID":"CA-2016-152156","Order Date":"08-11-2016","Ship Date":"11-11-2016",
+    "Ship Mode":"Second Class","Customer Name":"Claire Gute","Segment":"Consumer",
+    "City":"Henderson","State":"Kentucky","Region":"South","Category":"Furniture",
+    "Sub-Category":"Chairs","Sales":731.94,"Quantity":3,"Discount":0.0,"Profit":219.582
+  },
+  // etc
+]
+```
 
-<script type="text/html" class="step1dataophtml" src="snippets/example-output.html"></script>
+You could also visit
+[http://localhost:9988/tutorials/data?_limit=10&_format=html](http://localhost:9988/tutorials/data?_limit=10&_format=html)
```

Comment:
Use relative URL maybe? If we want this to accessible from https://learn.gramener.com/guide/

Reply:
The localhost URLs are kept like this on purpose, we want the user to actually see things on their own machines.